synthetics/browser-test/browser-test-metrics.rst

```diff
@@ -67,24 +67,24 @@ Test-level metrics in Browser tests
 
    * - Run count
      - ``synthetics.run.count``
-     - Total number of runs for the test. This metric contains dimensions such as ``success: true`` and ``failed: false`` to indicate whether the run succeeded or failed. 
+     - Total number of runs for the test. This metric contains dimensions such as ``success: true`` and ``failed: false`` to indicate whether the run succeeded or failed.
 
    * - Run-level duration
      - ``synthetics.run.duration.time.ms``
-     - The total duration of the entire run, including all pages and synthetic transactions. 
+     - The total duration of the entire run, including all pages and synthetic transactions.
+
 
-  
 .. _page-level-metrics:
 
 Page-level metrics in Browser tests
 --------------------------------------
-Browser tests in Splunk Synthetic Monitoring automatically capture a set of 46 default metrics for each page load. These metrics are grouped into several categories. See the tables below for details on all default page-level metrics. 
+Browser tests in Splunk Synthetic Monitoring automatically capture a set of 45 default metrics for each page load. These metrics are grouped into several categories. See the tables below for details on all default page-level metrics.
```

Comment:
I just noticed that we say "40+" metrics at the top of this page -- maybe we should do the same here to give us some buffer if we ever add a new metric 👀